### DISCUSSION TITLE
Unable to include a reference to a video

### DISCUSSION THREAD

**Discussion by johnmuchow** [quarto-dev/quarto-cli#5846] [Q&A]
### Description

I am using the format {{< video local-video.mp4 >}} to include a video that is in the same folder as the .qmd file. I get the error message "The media could not be loaded, either because the server or network failed or because the format is not supported."

The file is an mp4. Is there any additional debug information available to help narrow down the issue?

**Comment by cscheid** [comment]
Thanks for the report. We need a full reproducible example to understand better what's going on.

**Reply by johnmuchow** [reply]
Fair enough. If I can get a simple example working, should I post the code here?

**Reply by cscheid** [reply]
Yes, absolutely. It's just that it's too hard for us to speculate on too many variables if we don't have a single .qmd (or a small project) we can look at ourselves. Thank yoU!

**Comment by ghost** [comment]
Thanks for reporting.  I've [moved it to this issue](https://github.com/quarto-dev/quarto-cli/issues/5847).
